output-comparator/src/main/java/org/mobilitydata/gtfsvalidator/outputcomparator/model/report/AffectedSource.java

```diff
@@ -0,0 +1,26 @@
+package org.mobilitydata.gtfsvalidator.outputcomparator.model.report;
+
+import com.google.auto.value.AutoValue;
+
+/**
+ * A dataset source that had a different number of validation errors of a particular type in the
+ * comparison set.
+ */
+@AutoValue
+public abstract class AffectedSource {
+
+  public abstract String sourceId();
+
+  public abstract String sourceUrl();
+
+  /**
+   * The number of notices of a particular type.
+   *
+   * @return
+   */
+  public abstract int count();
```

Comment:
When you say "the number of notices of a particular type", this should return the number of errors or warning, for instance? 

I'm not the most familiar with Autovalue, but won't the method `create` below assign `totalNotices` to `count`? Or will the library will create a new variable `self.totalNotices`? I'm just wondering what is the expected behaviour between `count` and `totalNotices`.

Reply:
I think it would always be the number of errors in practice based on the calling methods.  We could clarify that here.

As for @AutoValue behavior, it will assigned `totalNotices` to `count`.  Internally, @AutoValue is generating a constructor that has a param for each defined field, in field-declaration order.  I'm not sure `count` is the most descriptive name, but I kept it to match the schema of the original JSON output.  We could probably change that without breaking too much if you thought it was appropriate?

Reply:
I mistook "type" for "severity" here. Since we count the errors by source, wouldn't it be "The number of notices of a particular severity" instead?

If by `type` we mean the `count` per notice, eg. the `count` for `missing_required_field`, I think it makes sense.

`totalNotices` is used in the report as the number of notices per type of notice, so if this is what we want to do here, it makes sense too. I think in both cases, we could just make the docstring more descriptive and that would do the trick.

> I'm not sure count is the most descriptive name, but I kept it to match the schema of the original JSON output.

Do you mean the original output of the acceptance tests? If yes, we can change `count` for `totalNotices` or another more descriptive name without any problem. It wouldn't be a breaking change, only the validation report must stay the same.

Reply:
It's definitely the notice type, not the severity.  I renamed the field + tried to reword the comment for clarity.

Reply:
Perfect, thanks!